js/main.js

```diff
@@ -10,16 +11,95 @@ $photoInput.addEventListener('input', function (event) {
 });
 
 const $handleForm = document.querySelector('.form');
+const $list = document.querySelector('.no-bullets');
+const $entryForm = document.querySelector('.entry-form');
+const $entries = document.querySelector('.entries');
 
 $handleForm.addEventListener('submit', function (event) {
   event.preventDefault();
-  data.entries.unshift({
+  const newObject = {
     title: $titleInput.value,
     url: $photoInput.value,
     notes: $notesInput.value,
     entryId: data.nextEntryId,
-  });
-  data.nextEntryId = data.nextEntryId + 1;
+  };
+  data.entries.unshift(newObject);
+  const $newObject = renderEntry(newObject);
+  $list.prepend($newObject);
+  viewSwap('entries');
+  toggleNoEntries();
+  data.nextEntryId = data.nextEntryId++;
   $handlePhoto.src = 'images/placeholder-image-square.jpg';
   $handleForm.reset();
 });
+
+// Issue #2
+function renderEntry(entry) {
+  const $li = document.createElement('li');
+  const $imageDiv = document.createElement('div');
+  const $image = document.createElement('img');
+  const $textDiv = document.createElement('div');
+  const $h3 = document.createElement('h3');
+  const $p = document.createElement('p');
+
+  $li.setAttribute('class', 'row');
+  $imageDiv.setAttribute('class', 'column-half');
+  $image.setAttribute('src', entry.url);
+  $image.setAttribute('alt', 'Fix your link!');
+  $textDiv.setAttribute('class', 'column-half');
+
+  $h3.textContent = entry.title;
+  $p.textContent = entry.notes;
+
+  $li.appendChild($imageDiv);
+  $li.appendChild($textDiv);
+  $imageDiv.appendChild($image);
+  $textDiv.appendChild($h3);
+  $textDiv.appendChild($p);
+
+  return $li;
+}
+
+document.addEventListener('DOMContentLoaded', function (event) {
+  for (let i = 0; i < data.entries.length; i++) {
+    const $newEntry = renderEntry(data.entries[i]);
+    $list.prepend($newEntry);
+  }
+  viewSwap(data.view);
+  if ($list.children.length > 1) {
+    toggleNoEntries();
+  }
```

Comment:
`if` statement isn't needed, we are already doing the check inside of `toggleNoEntries`